### PR TITLE
Add better workaround for text backend test GPU fix

### PR DESCRIPTION
**Pull request by kyleabeauchamp**

### CODE REVIEW

File: pymc3/tests/backend_fixtures.py [hunk 1]
```diff
@@ -286,8 +287,10 @@ def test_len(self):
 
     def test_dtypes(self):
         for varname in self.test_point.keys():
-            assert self.expected[0][varname].dtype == \
-                             self.mtrace.get_values(varname, chains=0).dtype
+            assert dtype_equals_modulo_precision(
+                self.expected[0][varname].dtype,
+                self.mtrace.get_values(varname, chains=0).dtype
+                    )
```
Comment (twiecki) [inline pymc3/tests/backend_fixtures.py]: the brackets are mis-aligned.

**Comment by kyleabeauchamp** [comment]
fixed

**Comment by twiecki** [comment]
Actually, does this not hide the problem in the backend? I feel like that should respect dtype.

**Comment by kyleabeauchamp** [comment]
Is it a design choice that the GPU precision and the storage precision should always be equal?  I know of lots of numerical tools that allow separate configuration of both precisions.

**Comment by kyleabeauchamp** [comment]
Closing in favor of #2081.  IMHO either this or #2081 would be a reasonable approach, but there is no need to merge both.

**Pull request closed without merging**